schema.graphql

```diff
@@ -110,10 +110,14 @@ type SavedItem implements RemoteEntity @key(fields: "id") @key(fields: "url") {
   """
   item: ItemResult!
   """
-  The Tags assocaited with this SavedItem
+  The Tags associated with this SavedItem
   """
   tags: [Tag!]
   """
+  The Suggested Tags associated with this SavedItem
+  """
+  suggestedTags: [Tag!]
```

Comment:
should we update the list graph in the spec repo?

Reply:
Can you link me? I search that repo and struggle in it.

Reply:
https://github.com/Pocket/spec/blob/main/source/client-api/list/schema.graphql

Reply:
https://github.com/Pocket/spec/pull/137